### PR TITLE
Update JGDAS ENKF ECEN job 

### PR DESCRIPTION
**Pull request by mingshichen-noaa**
# Description
 
NCO has requested that each COM variable specify whether it is an input or an output. This completes that process for the global jgdas enkf ecen job.

Refs https://github.com/NOAA-EMC/global-workflow/issues/2451

# Type of change

- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
 
- Clone and build on RDHPCS
- Cycled tests on Hercules
- Forecast-only tests on Hercules
 
# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by CoryMartin-NOAA — COMMENTED** [review]
Why are there no changes for `COM_ATMOS_ANALYSIS_STAT`?

**Review by KateFriedman-NOAA — COMMENTED** [review]
I second @CoryMartin-NOAA 's comment. Changes in this PR look good so far but there are other `COM` variables in `scripts/exgdas_enkf_ecen.sh` that also need to be updated to their `COMIN`/`COMOUT` versions. Please update those variables as well, thanks!

**Comment by mingshichen-noaa** [comment]
@CoryMartin-NOAA @KateFriedman-NOAA 
Hi both,
I am sorry to have missed git  push. Right now I have made updating.

**Comment by KateFriedman-NOAA** [comment]
> I am sorry to have missed git push. Right now I have made updating.

@mingshichen-noaa Thanks for committing those additional updates. Question, the testing you did included these `COM` changes that you just pushed?

**Comment by mingshichen-noaa** [comment]
Kate,
Yes, testing and git push were done yesterday.

File: scripts/exgdas_enkf_ecen.sh
Comment (KateFriedman-NOAA) [inline scripts/exgdas_enkf_ecen.sh]: Also please double check lines 179 and 181. I think they should also be `COMOUT_*`. Thanks!

File: scripts/exgdas_enkf_ecen.sh
Comment (KateFriedman-NOAA) [inline scripts/exgdas_enkf_ecen.sh]: I think lines 159 and 161 should be `COMOUT_*`. Please double check, thanks!

File: jobs/JGDAS_ENKF_ECEN
Comment (KateFriedman-NOAA) [inline jobs/JGDAS_ENKF_ECEN]: See my comments in the other script. I think this should be `COMOUT_ATMOS_ANALYSIS_STAT`. Please double check, thanks!

**Review by KateFriedman-NOAA — CHANGES_REQUESTED** [review]

**Comment by CoryMartin-NOAA** [comment]
I agree with @KateFriedman-NOAA those are links to files produced by executables and thus should be COMOUT

**Comment by mingshichen-noaa** [comment]
Kate,
I agree with your comments and suggestions. I have made change and done git add/commit/push them.

**Review by KateFriedman-NOAA — APPROVED** [review]
I think the changes are correct now. Approve pending successful completion of CI tests and approval by other reviewers.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Review by DavidHuber-NOAA — APPROVED** [review]

**Pull request merged**
